addons/api/addon/models/role.js

```diff
@@ -74,6 +74,34 @@ export default class RoleModel extends GeneratedRoleModel {
 
   // =methods
 
+  /**
+   * Saves grant scope ids on the role via the `set-grant-scopes` method.
+   * @param {[string]} grantScopeIDs
+   * @return {Promise}
+   */
+  setGrantScopes(grantScopeIDs) {
+    return this.save({
+      adapterOptions: {
+        method: 'set-grant-scopes',
+        grantScopeIDs,
+      },
+    });
+  }
+
+  /**
+   * Removes grant scope ids on the role via the `remove-grant-scopes` method.
+   * @param {[string]} grantScopeIDs
+   * @return {Promise}
+   */
+  removeGrantScopes(grantScopeIDs) {
```

Comment:
When would we call `removeGrantScopes` instead of calling `setGrantScopes`?

Reply:
Looking at our designs, we won't need a dedicated `removeGrantScopes` anymore. I will remove this.